frontend/src/CourseCard.css

```diff
@@ -0,0 +1,47 @@
+.CourseCard {
+    display: flex;
+    background: #F6F8F5;
+    box-shadow: 0px 4px 25px rgba(0, 0, 0, 0.24);
+    border-radius: 15px;
+    font-family: 'Roboto', sans-serif; 
+}
+
+.CardBody {
+    align-self: center;
```

Comment:
I noticed you use `align-self: center` for both `CardBody` and `CardButton` (code duplication), wouldn't it be simpler to use `align-items` on the parent element and set it to `center`? ;) Not sure how you feel about Flexbox in general, but for me, what helped a lot was to play [this](https://geddski.teachable.com/p/flexbox-zombies) game. It was entertaining and educational. Loved it. You have to try it.

Reply:
That's true! CSS is really not my favorite but thanks for sharing this link, maybe I'll start loving it after playing this game !